bessctl/conf/samples/nat.bess

```diff
@@ -0,0 +1,28 @@
+import scapy.all as scapy
```

Comment:
Would you be able to add comments both to the head of this script and to a few of the commands that are not commented explaining their intent / function?